script/deploy/deployAndStoreCREATE3Factory.sh

```
@@ -0,0 +1,83 @@
+  local FILE_SUFFIX=$(getFileSuffix "$ENVIRONMENT")
+
+  # logging for debug purposes
+  echo ""
+  echoDebug "in function deployAndStoreCREATE3Factory"
+  echoDebug "NETWORK=$NETWORK"
+  echoDebug "ENVIRONMENT=$ENVIRONMENT"
+  echoDebug "FILE_SUFFIX=$FILE_SUFFIX"
+  echo ""
+
+  # deploy CREATE3Factory
+  local PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)